SpringProject/src/main/java/com/gradproject2019/Conference.java

```diff
@@ -1,14 +1,27 @@
+package com.gradproject2019;
+
+import javax.persistence.*;
 import java.time.Instant;
 
+@Entity
```

Comment:
Put the table name in here as "conferences" otherwise it will generate as "conference", I have a look in your db migration V1 and you will also need to change the table name to conferences there. https://stackoverflow.com/questions/29087626/entity-class-name-is-transformed-into-sql-table-name-with-underscores

Reply:
We are a bit unclear on this, should we rename the Conference class?

Reply:
Nope the Conference class will remain singular